### DISCUSSION TITLE
怎么样把多个不同类型的网盘挂载到一个目录中?

### DISCUSSION THREAD

**Discussion by dfaofeng** [libsgh/PanIndex#177] [General]
![image](https://user-images.githubusercontent.com/14891398/141954160-307d93b4-33db-4c66-a5b8-1bd202758219.png)
类似与1是阿里的,2是od的

**Comment by libsgh** [comment]
挂载阿里和od两个网盘，后台改下显示配置
![image](https://user-images.githubusercontent.com/25656088/141957883-10dcb44d-aaf1-410c-a160-41bd7832da35.png)
效果：
![image](https://user-images.githubusercontent.com/25656088/141957972-d1cb4147-18e7-423c-9f8f-bf5bb7706bac.png)

**Reply by dfaofeng** [reply]
![image](https://user-images.githubusercontent.com/14891398/142137958-44feaa58-58b6-4a80-b4fa-db7ad1e1bcab.png)
为什么我设置每分钟同步,只生效一次呢?
